test/base/ByteBufferTest.ooc

```diff
@@ -9,20 +9,6 @@
 use base
 use unit
 
-CustomAllocator: class extends AbstractAllocator {
```

Comment:
Yes, the allocation goes through an external library call. We could allocate aligned buffers ourselves, but I think there is also some internal caching in that library.